### PR TITLE
Adapt asserts to changed scenario

### PR DESCRIPTION
**Pull request by palango**
## Description

Fixes: https://github.com/raiden-network/raiden/issues/5460

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Documentation
    - [ ] A new CLI flag was introduced, is there documentation that explains usage?
- Specs
    - [ ] If this is a protocol change, are the specs updated accordingly? If so, please link PR from the specs repo.
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

File: raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml [hunk 1]
```diff
@@ -55,11 +55,11 @@ scenario:
             - wait_blocks: 1
             # Assert that correct amount was tranferred
             - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 999_000_000_000_000_000}
-            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_001_000_000_000_000_000}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000}
```
Comment (Dominik1999) [inline raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml]: You need to set `total_deposit: 1_000_000_000_000_000_000` to `total_deposit: 0`, right?

File: raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml [hunk 2]
```diff
@@ -55,11 +55,11 @@ scenario:
...
             - assert: {from: 1, to: 2, total_deposit: 1_000_000_000_000_000_000, balance: 999_000_000_000_000_000}
-            - assert: {from: 2, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_001_000_000_000_000_000}
+            - assert: {from: 2, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000}
```
Comment (Dominik1999) [inline raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml]: same here

File: raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml [hunk 3]
```diff
@@ -55,11 +55,11 @@ scenario:
...
             - assert: {from: 2, to: 3, total_deposit: 1_000_000_000_000_000_000, balance: 999_000_000_000_000_000}
-            - assert: {from: 3, to: 2, total_deposit: 1_000_000_000_000_000_000, balance: 1_001_000_000_000_000_000}
+            - assert: {from: 3, to: 2, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000}
```
Comment (Dominik1999) [inline raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml]: same here

File: raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml (hunk 1 above)
Comment (palango) [reply raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml]: Good catch 👀

**Pull request merged**